src/components/pages/AdminTools/ManageUsers.js

```diff
@@ -1,11 +1,21 @@
 import React, { useEffect, useState } from 'react';
-import axios from 'axios';
+import axiosWithAuth from '../../../utils/axiosWithAuth';

```

Comment:
Wow, replacing the axios calls with axiosWithAuth cleans up the code a bunch! Good call.